### ISSUE TITLE
take out Ctrl-Alt-Del from the example

### ISSUE THREAD

**Issue by StefanL38** [T-vK/ESP32-BLE-Keyboard#122]
take out Ctrl-Alt-Del from the example

this is a really bad idea!

because uploading a new code-version without it is rellay hard!

**Comment by T-vK** [comment]
Turning off Bluetooth shoukd do the trick. I'm open to having it removed from the example if anyone is willing to make a Pull Request for that. It would have to be removed from the README as well btw.

**Comment by ScaredyCat** [comment]
I have to agree that it shouldn't be there in an example, or at least comment out that bit so people can see the multiple modifiers in code. Examples shouldn't intentionally look to 'harm' users.

**Comment by T-vK** [comment]
Excuse me, "intentionally harm"?! This shortcut is supposed to open the Task Manager in Windows.

**Comment by ScaredyCat** [comment]
You don't know what the target OS the user is using.

**Comment by T-vK** [comment]
Do you even know what the word intention means?

**Comment by ScaredyCat** [comment]
I'll admit it's badly worded but #10 and #62 were already known existing problems. Apologies if I caused offence with my phrasing, none was intended.
